tools/install.sh

```diff
@@ -0,0 +1,668 @@
+# usage() {
+#     cat 1>&2 <<EOF
+# rustup-init 1.25.1 (48d233f65 2022-07-12)
+# The installer for rustup
+# 
+# USAGE:
+#     rustup-init [FLAGS] [OPTIONS]
+# 
+# FLAGS:
+#     -v, --verbose           Enable verbose output
+#     -q, --quiet             Disable progress output
+#     -y                      Disable confirmation prompt.
+#         --no-modify-path    Don't configure the PATH environment variable
+#     -h, --help              Prints help information
+#     -V, --version           Prints version information
+# 
+# OPTIONS:
+#         --default-host <default-host>              Choose a default host triple
+#         --default-toolchain <default-toolchain>    Choose a default toolchain to install
+#         --default-toolchain none                   Do not install any toolchains
+#         --profile [minimal|default|complete]       Choose a profile
+#     -c, --component <components>...                Component name to also install
+#     -t, --target <targets>...                      Target name to also install
+# EOF
+# }
+
+usage() {
+    cat 1>&2 <<EOF
+manta-cli-init 1.0.0-beta
+The installer for Manta Network CLIs
+
+USAGE:
+    manta-cli-init [FLAGS] [OPTIONS]
+
+FLAGS:
+    -v, --verbose           Enable verbose output
+    -q, --quiet             Disable progress output
+    -h, --help              Prints help information
+    -V, --version           Prints version information
+EOF
+}
+
+main() {
+    printf "Manta CLI Tools\n"
+
+    downloader --check
+    need_cmd uname
+    need_cmd mktemp
+    need_cmd chmod
+    need_cmd mkdir
+    need_cmd rm
+    need_cmd rmdir
+
+    get_architecture || return 1
+    local _arch="$RETVAL"
+    assert_nz "$_arch" "arch"
+
+    case "$_arch" in
+        aarch64-apple-darwin | x86_64-apple-darwin | x86_64-pc-windows-msvc | x86_64-unknown-linux-gnu)
+            ;;
+        *)
+            echo "Using unsupported target ${_arch}. Please build the binaries from source at https://github.com/Manta-Network/manta-rs/tree/main/manta-trusted-setup."
+            return 1
+            ;;
+    esac
+
+    local _ext=""
+    local _dir=""
+    case "$_arch" in
+        *windows*)
+            _ext=".exe"
+            _dir="."
+            printf "[WARN] Unable to install windows distribution automatically: copying binaries to the current directory instead.\n"
+            ;;
+        *)
+            _dir="$HOME/.manta/bin"
+            ;;
+    esac
+
+    local _url="https://github.com/manta-network/manta-rs/releases/download/v0.5.5/manta-trusted-setup-${_arch}${_ext}"
+
+    printf "[INFO] Installing manta-trusted-setup for the ${_arch} target ... "
+
+    local _file="${_dir}/manta-trusted-setup${_ext}"
+
+    ensure mkdir -p "$_dir"
+    ensure downloader "$_url" "$_file" "$_arch"
+    ensure chmod u+x "$_file"
+
+    printf "DONE.\n"
+
+    case ":${PATH}:" in
+        *:"${_dir}":*)
+            ;;
+        *)
+            echo "\nexport PATH=\"${_dir}:\$PATH\";" >> "$HOME/.profile"
```

Comment:
on my system this produces the following error:
```
$ source ~/.profile
bash: nexport: command not found...
```
which looks like a terminal quoting issue to me. a simple fix would be to add a single `echo` (no args) command on the line above which would produce the newline, then remove the `\n` from the start of this echo.

Reply:
eg:
```bash
echo >> "$HOME/.profile"
echo "export PATH=\"${_dir}:\$PATH\";" >> "$HOME/.profile"
```